koko_hardware:
    joint_names: ['base_roll_joint']
    motor_names: ['base_roll_motor']
    motor_ids: [1]
    gear_ratio: [7.1875] # 8.24448529412 results from 7.1875 * (39 / 34) from the output gear
    current_slope: [0.626] # 1.0 is decent approx for small motors
    current_offset: [0.0]
    calibration_initial_angles: [0.0]
    hardstop_start_angles: [0.0] # angles that you the robot is in for hardstop calibration
    directions:        [1.0]
    torque_directions: [1.0]
    joint_torque_directions: [1.0]
    min_angles: [-8.2]
    max_angles: [8.2]
    calibration_encoder: [3.5856800916826863] # deprecated
    calibration_joint_vive: [0.085327798760949689] #deprecated
    paired_constraints: []
    #vive or rviz
    target_mode: "vive"
    softstop_torque_limit: 0.0
    softstop_tolerance: 0.0
    endlink: "shoulder_link"
